backend/tests/integration/search/test_api.py

```diff
@@ -30,3 +30,97 @@ def test_literature_get_records_by_pids_returns_correct_record(
     assert len(result) == len(expected_control_numbers)
     for rec in result:
         assert rec.to_dict()["control_number"] in expected_control_numbers
+
+
+def test_empty_literature_search(api_client, db, es_clear, create_record):
+    create_record("lit")
+    create_record("lit")
+    response = api_client.get("api/literature")
+
+    expectet_results_count = 2
```

Comment:
```suggestion
    expected_results_count = 2
```
(this typo is present everywhere)